debian/copyright

```diff
@@ -2,18 +2,380 @@ Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Contact: Odoo <info@odoo.com>
 
 Files: *
-Copyright: (C) 2004-2015 Odoo SA. (www.odoo.com)
+Copyright: Copyright (C) 2004-2015 Odoo SA. (www.odoo.com)
 License: LGPL-3+
- This program is free software: you can redistribute it and/or modify
- it under the terms of the GNU Lesser General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- (at your option) any later version.
+
+Files: odoo/tools/appdirs.py
+Copyright: Copyright (c) 2005-2010 ActiveState Software Inc
+           Copyright (c) 2013 Eddy Petrișor
+License: MIT
+
+Files: odoo/tools/lru.py
+Copyright: Copyright 2003 Josiah Carlson
+License: GPL-2
```

Comment:
Hmm, can this GPL lib be used by the LGPL core? 🤔

Reply:
Actually it was published by the author himself under PSF (BSD-style) on this activestate recipe: http://code.activestate.com/recipes/252524-length-limited-o1-lru-cache-implementation/
(Even if it was initially written for his [PyPE editor](https://sourceforge.net/projects/pype/), which is GPL-2)
We should fix the `debian/copyright` and mention the correct license next to the code.